contribs/gmf/src/directives/displayquerygrid.js

```diff
@@ -287,6 +262,39 @@ gmf.DisplayquerygridController = function($injector, $scope, ngeoQueryResult,
   this.unregisterSelectWatcher_ = null;
 };
 
+/**
+ * Init the controller
+ */
+gmf.DisplayquerygridController.prototype.$onInit = function() {
+  const featuresStyle = this['featuresStyleFn']();
```

Comment:
As a side note, all these brackets looks bad though they were not introduced by your PR.